modules/contracts/src.sol/AssetTransfer.sol

```diff
@@ -72,15 +66,17 @@ contract AssetTransfer is CMCCore, IAssetTransfer {
   }
 
   function _getOwnERC20Balance(address assetId) external view onlySelf returns (uint256) {
-    return IERC20(assetId).balanceOf(address(this));
+    return IERC20(assetId).balanceOf{gas: ERC20_BALANCEOF_GAS_LIMIT}(address(this));
```

Comment:
it seems in the majority of this PR we've replaced the calls to `IERC20` with calls to `LibERC20`. these changes are really only needed for transfers, but we may want to move this call in there as well for consistency

Reply:
Sure, we can do that. Right now, the `LibERC20` only wraps the calls that behave differently in standard-compliant and missing-return-value-bug tokens, so it is consistent in a way. But I get your drift and am also fine with the suggested alternative.

Reply:
Actually, on second thought, I'd prefer to leave it as it is, because we cannot restrict the amount of gas passed on for an _internal_ call, so we'd have to pass it along as a parameter and then have a special `balanceOf` function in the `LibERC20` that takes a gas parameter. That would be a bit ugly, and in the end, the overhead doesn't seem worth it. You're ok with that, @LayneHaber?

Reply:
Yeah sg, will approve + merge 👍